src/webpack/server.js

```diff
@@ -1,30 +1,30 @@
+const CONFIG = require('../constants/config')
 const https = require('https')
 const state = require('../proxyServer/state')
 const fs = require('fs')
-process.env['NODE_TLS_REJECT_UNAUTHORIZED'] = 0
-
 let isInit = false
 
 const init = function() {
   if (isInit) return
 
   isInit = true
 
-  const { localSSLDir, webpackOutputPath } = state.get()
+  const { webpackOutputPath, cert } = state.get()
 
-  const key = fs.readFileSync(`${localSSLDir}localdev-ssl.key`, 'ascii')
-  const cert = fs.readFileSync(`${localSSLDir}localdev-ssl.crt`, 'ascii')
+  require('ssl-root-cas')
```

Comment:
The guy that made this lib, is a fucking brilliant dev. And he writes the kind of explanations that took my 50 hour problem down to 5 mins. 

Respect!